tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-product-filter-test.php

```
@@ -0,0 +1,1117 @@
+		add_filter( 'wc_stripe_agentic_commerce_product_filter', $capture_filter );
+
+		try {
+			$filter->get_filters();
+			$this->assertSame( 1, $call_count );
+
+			$filter->get_filters();
+			$this->assertSame( 1, $call_count );
+		} finally {
+			remove_filter( 'wc_stripe_agentic_commerce_product_filter', $capture_filter );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_get_filters_only_calls_filter_once is not named in camelCase. (undefined)

(CamelCaseMethodName)